doc/classes/MultiMeshInstance2D.xml

```diff
@@ -4,7 +4,7 @@
 		Node that instances a [MultiMesh] in 2D.
 	</brief_description>
 	<description>
-		[MultiMeshInstance2D] is a specialized node to instance a [MultiMesh] resource in 2D.
+		[MultiMeshInstance2D] is a specialized node to instance a [MultiMesh] resource in 2D. This can be faster to render compared to displaying many [Sprite2D] nodes with large transparent areas, especially if the nodes take up a lot of space on screen at high viewport resolutions. This is because using a mesh designed to fit the sprite's opaque areas will reduce GPU fill rate utilization (at the cost of increased vertex processing utilization).
```

Comment:
```suggestion
		[MultiMeshInstance2D] is a specialized node to instance a [MultiMesh] resource in 2D. This can be faster to render compared to displaying many [Sprite2D] nodes with large transparent areas, especially if the nodes take up a lot of space on screen at high viewport resolutions. This is because using a mesh designed to fit the sprites' opaque areas will reduce GPU fill rate utilization (at the cost of increased vertex processing utilization).
```
Missed last time